src/components/Landing/Hero.tsx

```diff
@@ -35,17 +36,20 @@ const HeroCard = ({ wiki }: HeroProps) => {
         />
         <Flex p="3" align="center" gap={4}>
           <NextLink href={`/account/${wiki?.user?.id}`} passHref>
-            <DisplayAvatar address={wiki?.user?.id} />
+            <Avatar size="xs" src={avatar} />
           </NextLink>
           <Flex
             direction="column"
             justify="space-between"
             fontWeight="semibold"
           >
             <chakra.span>{wiki?.title}</chakra.span>
-            <chakra.span color="blue">
-              {shortenAccount(wiki?.user?.id || '')}
-            </chakra.span>
+
+            <NextLink href={`/account/${wiki?.user?.id}`} passHref>
+              <chakra.span color="blue">
```

Comment:
FOr accessibility issues, the child receiving `passHref` should be an `a` tag. 
```suggestion
            <NextLink href={`/account/${wiki?.user?.id}`} passHref>
              <chakra.a color="blue">
```